### ISSUE TITLE
 `openneuro login` should use `getpass(3)`

### ISSUE THREAD

**Issue by kousu** [OpenNeuroOrg/openneuro#1696]
**Describe the bug**

When doing `openneuro login`, the auth token is echoed back to the user. It's good practice to censor this, doubly-so since this auth token grants full access to one's account if copied into a browser.

**To Reproduce**

1. Run `openneuro login`
2. Enter your credentials
3. See them still on-screen

**Expected behavior**

You should be able to paste in your token without seeing it.

**Screenshots***

```
$ openneuro login
? Choose an OpenNeuro instance to use. https://openneuro.org/
? Enter your API key for OpenNeuro (get an API key from https://openneuro.org/keygen) thisismyjwthere
```

**Desktop (please complete the following information):**
- [x] Not applicable 

**Smartphone (please complete the following information):**
- [x] Not applicable 

**Additional context**

I don't the credentials just to be erased from the screen, I want them to never have been printed there in the first place. [node-getpass](https://www.npmjs.com/package/getpass) is a effective copy of [getpass(3)](https://linux.die.net/man/3/getpass) which enables this.

**Comment by nellh** [comment]
Thanks for the bug report, a fix for this was released in 3.18.0

**Comment by kousu** [comment]
Thank you for your work, Nell!
